tlsn/tlsn-notary/src/lib.rs

```diff
@@ -88,6 +100,7 @@ where
         Self { config, mux }
     }
 
+    /// Runs the notary server
```

Comment:
let's s/server/instance since this will perform only 1 notarization and then exit